include/glow/Backends/Backend.h

```diff
@@ -42,6 +42,9 @@ class Backend {
 
   /// Generate code for input function \param F.
   virtual std::unique_ptr<CompiledFunction> compile(Function *F) const = 0;
+  /// Generate code for input function \param F but do not collect constants.
+  virtual std::unique_ptr<CompiledFunction>
+  compileWithoutConstants(Function *F) const = 0;
```

Comment:
where is it invoked?

Reply:
Currently nowhere but it will be called from the Provisioner.